app/policies/organization_policy.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+# Policies for OrganizationsController
+class OrganizationPolicy < ApplicationPolicy
+  def initialize(_current_user = nil, _resource = nil)
+    super
+  end
+
+  def show?
+    true
+  end
+end
```

Comment:
update: current user is organization-admin
destroy: current user is organization-admin